### ISSUE TITLE
Add support for nested sorting

### ISSUE THREAD

**Issue by MasterEric** [Gudahtt/prettier-plugin-sort-json#15]
Currently the plugin does not sort the keys of JSON objects which are nested within arrays or within other objects.

Current behavior:

```
{
    "0": null,
    "a": null,
    "b": null,
    "exampleNestedObject": {
        "z": null,
        "a": null
    },
    "z": null
}
```

Desired behavior:
```
{
    "0": null,
    "a": null,
    "b": null,
    "exampleNestedObject": {
        "a": null,
        "z": null
    },
    "z": null
}
```

**Comment by mfulton26** [comment]
I would like this feature

**Comment by Gudahtt** [comment]
Thanks for the suggestion! This will be included in the next release, v0.0.2

**Comment by MasterEric** [comment]
@Gudahtt Excellent! Thank you for the hard work! :blush:
